src/daft-physical-plan/src/physical_planner/translate.rs

```diff
@@ -1099,6 +1102,26 @@ pub fn populate_aggregation_stages(
                     ));
                 final_exprs.push(col(second_stage_id).alias(output_name));
             }
+            AggExpr::ApproxDistinct(e) => {
+                let first_stage_id = agg_expr.semantic_id(schema).id;
+                let second_stage_id =
+                    AggExpr::MergeSketch(col(first_stage_id.clone()), SketchType::HyperLogLog)
+                        .semantic_id(schema)
+                        .id;
+                first_stage_aggs
+                    .entry(first_stage_id.clone())
+                    .or_insert(AggExpr::ApproxSketch(
+                        e.alias(first_stage_id.clone()),
+                        SketchType::HyperLogLog,
+                    ));
+                second_stage_aggs
+                    .entry(second_stage_id.clone())
+                    .or_insert(AggExpr::MergeSketch(
+                        col(first_stage_id).alias(second_stage_id.clone()),
+                        SketchType::HyperLogLog,
+                    ));
+                final_exprs.push(col(second_stage_id).alias(output_name));
+            }
```

Comment:
I don't think `sketches` are needed in the current algorithm

Reply:
I simply copied this from above - what should this look like?

Reply:
Also, where can I find documentation on sketches? It's not a public API

Reply:
HLL sketches are currently for internal use only (though I know some engines expose them). See https://en.wikipedia.org/wiki/HyperLogLog for more details

I think for us, since we're not doing any approximate counts, we can avoid the use of sketches and just call the aggregation functionality you defined elsewhere